sensor/common/sensor/central_communication_impl.go

```diff
@@ -380,7 +380,7 @@ func (s *centralCommunicationImpl) initialPolicySync(ctx context.Context,
 	if msg.GetNetworkBaselineSync() == nil {
 		return errors.Errorf("expected NetworkBaseline message but received %t", msg.Msg)
 	}
-	if err := detector.ProcessMessage(msg); err != nil {
+	if err := detector.ProcessMessage(context.TODO(), msg); err != nil {
```

Comment:
Would it make sense to pass on the function parameter context here ?

Reply:
That makes sense, I totally missed that. [bece180](https://github.com/stackrox/stackrox/pull/16064/commits/bece18029acb26ab1181f64c0d6212121aa7ffdf)